docs/resource-no-translation.md

```diff
@@ -0,0 +1,35 @@
+# resource-no-translation
```

Comment:
Add rule to main readme's pending release changelog

Reply:
updated